src/languages/es.js

```diff
@@ -1086,7 +1086,7 @@ export default {
         visibility: 'Visibilidad',
         restrictedDescription: 'Sólo las personas en tu espacio de trabajo pueden encontrar esta sala',
         privateDescription: 'Sólo las personas que están invitadas a esta sala pueden encontrarla',
-        publicDescription: 'Cualquiera puede encontrarlo',
+        publicDescription: 'Cualquier persona puede unirse a este grupo',
```

Comment:
We're using the term `sala` for `room` in `es.js` :) 

```suggestion
        publicDescription: 'Cualquier persona puede unirse a esta sala',
```